job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/ScheduledExecutorConfig.java

```diff
@@ -118,6 +118,22 @@ public static int getDropTablesSchedulerPeriod() {
         return Integer.parseInt(period);
     }
 
+    public static String getQueueType() {
```

Comment:
Update the readme with these new env's and their defaults.
https://github.com/JobService/job-service/blob/develop/job-service-scheduled-executor/README.md